test/regressions/test_operation_on_device_1.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+class TestOperationOnDevice1(TestCase):
+    def test_sum_on_device1(self, dtype=torch.float):
+        if torch.xpu.device_count() >= 2:
```

Comment:
For single card node, this test will be skipped by default, right?

Reply:
Yes.